### ISSUE TITLE
Add Sessions to the searched results

### ISSUE THREAD

**Issue by shreyasnbhat** [fossasia/open-event-droidgen#1826]
**Actual Behaviour**
This doesn't exist now.

**Expected Behaviour**
This should be present within the search results

**Would you like to work on the issue?**
Yes

**Target Milestone to solve the issue**
Milestone 11

@arp95 @the-dagger @iamareebjamal @Shailesh351 Should we add only the bookmarked sessions or all the sessions? Adding all the sessions would clutter our search activity. What I can do is place the Bookmarked Sessions on the top above the Track Results. In another PR I can add the addition of the result filter i.e what all type of results does the user want.

**Comment by Shailesh351** [comment]
@shreyasnbhat showing only bookmarked sessions don't make sense. IMO all the sessions should be shown in GLOBAL search results.

**Comment by shreyasnbhat** [comment]
@Shailesh351 Let's hear from others. I know that keeping all the sessions in the global search makes sense.

**Comment by mahikaw** [comment]
Agreed with @Shailesh351 All the sessions in the search are needed, otherwise there won't be any reason to search, because bookmarks can already be seen separately in the home page.

**Comment by shreyasnbhat** [comment]
@mahikaw @Shailesh351 Alright. Makes sense. 
This issue needs to be taken care of after #1810 is merged.

**Comment by shreyasnbhat** [comment]
@Shailesh351 @mahikaw @arp95 @iamareebjamal  For the addition of session what all shall I search within each session? Just the title or the title and some other attributes such as the abstract?

**Comment by mahikaw** [comment]
I think just the titles should do. Abstract will clutter the search it a bit @shreyasnbhat

**Comment by iamareebjamal** [comment]
Only title should do
